package.json

```diff
@@ -65,7 +65,7 @@
     "done-serve": "^3.1.0",
     "donejs-travis-deploy-to-firebase": "^1.0.2",
     "donejs-travis-to-heroku": "0.0.2",
-    "place-my-order-api": "^1.4.0",
+    "place-my-order-api": "^1.3.0",
```

Comment:
Why was this changed?

Reply:
There's no `v1.4.0` https://www.npmjs.com/package/place-my-order-api